Yosemite/Yosemite/Stores/InAppPurchaseStore.swift

```diff
@@ -5,8 +5,11 @@ import Networking
 
 public class InAppPurchaseStore: Store {
     private var listenTask: Task<Void, Error>?
+    private let remote: InAppPurchasesRemote
+    private var useBackend = true
 
     public override init(dispatcher: Dispatcher, storageManager: StorageManagerType, network: Network) {
+        remote = InAppPurchasesRemote(network: network)
```

Comment:
Perhaps we could inject the remote so we can mock if required when testing?

Reply:
Looking at all the other stores in `AuthenticatedState`, I don't think we typically inject remotes, but instead allow injecting a mock `Network` to simulate network responses